privacyidea/lib/policy.py

```diff
@@ -798,50 +812,70 @@ def _policy_matches_request_header_condition(policy, key, comparator, value, req
                         u" is not available".format(policy["name"], key))
 
     @staticmethod
-    def _policy_matches_userinfo_condition(policy, key, comparator, value, user_object):
+    def _policy_matches_info_condition(policy, key, comparator, value, type, user_object=None, dbtoken=None):
```

Comment:
We cannot omit the type, since if both user_object and dbtoken are None, we could not tell whether to throw an exception or not. In this sense both paths behave a bit differently.

Reply:
I would think this might be somehow possible, but after all it does not matter, since it is only an internal function.
So I am fine with leaving it that way.

...for now! ;-)